nl-be/auth.json

```diff
@@ -0,0 +1,284 @@
+            "code_input_error_expired_code": "Je bevestigingscode is verlopen, vraag een nieuwe aan",
+            "didnt_receive_a_code_description": "Geen code ontvangen? ${didnt_receive_a_code_link}",
+            "didnt_receive_a_code_link": "Code opnieuw verzenden"
+        },
+        "sign_up_page": {
+            "page_title": "Aanmelden | ${business_name}",
+            "heading": "Maak een account aan",
+            "description": "Het is geweldig om u hier te hebben",
+            "first_name_input_label": "Voornaam",
+            "first_name_input_error_required": "Voer uw voornaam in",
+            "last_name_input_label": "Achternaam",
+            "last_name_input_error_required": "Voer uw achternaam in",
+            "email_input_label": "E-mail",
+            "email_input_error_required": "E-mail is verplicht",
+            "email_input_error_invalid": "Voer een geldig e-mailadres in",
+            "email_input_error_invitation_mismatch": "Het e-mailadres moet overeenkomen met het e-mailadres waarnaar uw uitnodiging is verzonden",
+            "continue_with_email_button": "Maak uw account",
+            "continue_with_social_provider_button": "Doorgaan met ${social_provider_name}",
+            "auth_block_separator": "Of",
+            "continue_with_enterprise_provider_button": "Doorgaan met ${enterprise_provider_name}",
+            "already_have_an_account_description": "Heet u al een account?",
+            "already_have_an_account_link": "Inloggen",
+            "auth_form_error_org_not_accepting_registrations": "De door u geselecteerde organisatie accepteert geen aanmeldingen",
+            "auth_form_error_no_auth_methods_enabled": "Sorry, we zien momenteel geen manier om u te verifiëren",
+            "auth_form_error_terms_not_accepted": "U moet de voorwaarden lezen en accepteren om verder te kunnen gaan",
+            "invitation_description": "U bent uitgenodigd om lid te worden van ${business_name}. Login of maak een account aan om de uitnodiging te accepteren.",
+            "disclaimer_clickwrap_input_label": "Voorwaarden accepteren",
+            "disclaimer_clickwrap_error_required": "U moet de voorwaarden lezen en accepteren om verder te kunnen gaan",
+            "disclaimer_dialog_heading": "Accepteren om door te gaan",
+            "disclaimer_dialog_description": "Om door te gaan met aanmelden, moet u onze voorwaarden lezen en accepteren",
+            "disclaimer": "Door verder te gaan, gaat u akkoord met onze voorwaarden",
+            "disclaimer_terms_of_use_link": "Gebruiksvoorwaarden",
+            "disclaimer_privacy_link": "Privacyverklaring"
+        },
+        "sms_verify_code_page": {
+            "page_title": "Multi-factor verificatie | ${business_name}",
+            "heading": "Telefoonnummer verifiëren",
+            "description": "Voer de code in die we zojuist hebben verzonden naar ${telefoon_nummer].",
```

Comment:
`]`

Reply:
`telefoon_nummer` has also been translated - this variable should be English

Reply:
@DaveOrDead updated